services/svc_googlephotos.py

```diff
@@ -192,16 +194,26 @@ def isGooglePhotosEnabled(self):
     '''
     return not (data['status'] == 403 and 'Enable it by visiting' in data['content'])
 
-  def getUrlFromImages(self, types, displaySize, images):
+  def getUrlFromImages(self, types, displaySize, images, randomize):
```

Comment:
Yes, obviously, given the previous comment, this will obvious need to be handled specially unless you can think of a way to abstract this behavior into the `base.py`

Reply:
I changed `getUrlFromImages` to a more abstract `selectImage` of `base.py`